Modules/_testmultiphase.c

```diff
@@ -820,16 +820,12 @@ static PyModuleDef_Slot meth_state_access_slots[] = {
 };
 
 static PyModuleDef def_meth_state_access = {
-    PyModuleDef_HEAD_INIT,                      /* m_base */
```

Comment:
We don't usually accept the style changing PR.

Reply:
I asked for this PR on purpose :-) Read comments starting at: https://github.com/python/cpython/pull/19936#pullrequestreview-405978698

Reply:
I see :)